css/style.css

```diff
@@ -1,9 +1,20 @@
-
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+:root {
+  --grey-dark: #363434;
```

Comment:
This is a really good use of CSS variables! Did you learn about them in the lesson?

Reply:
Not all of that I reading an extera book